.travis.yml

```diff
@@ -19,7 +19,9 @@ matrix:
         - php: hhvm
 
 before_install:
+    - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi;
     - composer self-update
+    - if [ "$GITHUB_OAUTH_TOKEN" ]; then composer config --global github-oauth.github.com ${GITHUB_OAUTH_TOKEN}; fi;
```

Comment:
is it? IIRC it's just to avoid hitting the API rate limit (you have a higher quota when authenticated, regardless of your permissions).

But yeah it's not an optional parameter, or rather we don't want it to be optional, that's why we don't a check IMO.